examples/nas/textnas/utils.py

```diff
@@ -14,7 +14,7 @@
 
 def get_length(mask):
     length = torch.sum(mask, 1)
-    length = length.long()
+    length = length.long().cpu()
```

Comment:
why "cpu()" is necessary, this is a bug fix?

Reply:
I think it's caused by PyTorch version upgrade. It reports some device misalignment error. I'm sure the problem was not there before.